spock-core/src/main/java/org/spockframework/runtime/Sputnik.java

```diff
@@ -51,6 +67,27 @@ public Sputnik(Class<?> clazz) throws InitializationError {
     this.clazz = clazz;
   }
 
+  @Override
+  protected List getChildren() {
+    throw new UnsupportedOperationException("workflow from ParentRunner not supported");
```

Comment:
Yes we have, but it is not enough. Because iteration can be reported independently we should fetch all iteration data before list of children will be generated.
I think we should do so because it will allow rerun single iteration from IDE. But it is hard, let's move this functionality to another pull-request.